app/assets/javascripts/client/inventories/InventoryReportDownloadCtrl.js

```diff
@@ -0,0 +1,31 @@
+(function() {
+  'use strict';
+  angular.module('PDRClient').controller('InventoryReportDownloadCtrl', InventoryReportDownloadCtrl);
+
+  InventoryReportDownloadCtrl.$inject = ['$scope', 'UrlService'];
+
+  function InventoryReportDownloadCtrl($scope, UrlService) {
+    var vm = this;
+    vm.inventory = $scope.inventory;
+    console.log("Report sidebar Inventory ", vm.inventory);
+    vm.download = function() {
+      if(vm.downloadOptions.productEntries) {
+        var link = angular.element('<a/>');
+        link.attr({
+          href: UrlService.url('/inventories/' + vm.inventory.id + '/product_entries.csv'),
+          target: '_blank',
+          download: 'inventory_' + vm.inventory.id + '_product_entries_report.csv'
+        })[0].click();
+      }
+      if(vm.downloadOptions.dataEntries) {
+        var link = angular.element('<a/>');
+        link.attr({
+          href: UrlService.url('/inventories/' + vm.inventory.id + '/data_entries.csv'),
+          target: '_blank',
+          download: 'inventory_' + vm.inventory.id + '_data_entries_report.csv'
+        })[0].click();
+      }
```

Comment:
There was a `/` after `'/inventories'`, the add operation was resulting in `NaN`, it was driving me crazy.